package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "jslab_frontend_part",
+  "version": "1.0.0",
+  "description": "set up frontend",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack serve --mode development",
+    "build": "webpack --mode development",
```

build is usually implemented in production mode
```suggestion
    "build": "webpack --mode production",
```